sanic_routing/router.py

```diff
@@ -367,51 +507,68 @@ def routes(self):
             [route for group in self.groups.values() for route in group]
         )
 
-    def optimize(self, src: t.List[Line]) -> None:
-        """
-        Insert NotFound exceptions to be able to bail as quick as possible,
-        and realign lines to proper indentation
-        """
-        offset = 0
-        current = 0
-        insert_at = set()
-        for num, line in enumerate(src):
-            if line.indent < current:
-                if not line.src.startswith("."):
-                    if offset < 0:
-                        offset += 1
-                    else:
-                        offset = 0
-
+    def _optimize(self, node) -> None:
+        warn(
+            "Router AST optimization is an experimental only feature. "
+            "Results may vary from unoptimized code."
+        )
```

Comment:
w00t. You really put some effort into this router.

Reply:
:upside_down_face: Still plenty more to go.